raviewer/__main__.py

```diff
@@ -66,7 +66,8 @@ def run(file_path, width, height, color_format, export, args):
             color_format, _ = classify_top1(img)
         if width == 0:
             width, _ = predict_resolution(img, color_format)[0]
-        img = parse_image(img.data_buffer, color_format, width)
```

Comment:
I'd rather not leave the endianness decision to the predictor mechanism by default - it is a heuristic that determines endianness based on certain patterns in the data, which may not always yield desired results. I'd prefer to stick to little endian as default.

A better approach would be to introduce a flag to arguments, e.g. `--endianness`, allowing to select between `little`, `big` and `auto`/`detect`, and leave the current default as is.

Reply:
Of course, will rework it then